packages/cli/src/commands/hydrogen/dev.ts

```diff
@@ -156,3 +170,85 @@ async function runDev({
     },
   });
 }
+
+interface VariablesArguments {
+  root: string;
+  shop?: string;
+}
+async function combinedEnvironmentVariables({root, shop}: VariablesArguments) {
```

Comment:
Can we move this logic to `lib`? Just to keep this file more focused on general dev logic. Also, this will probably be reused by the preview command at some point.

Reply:
[Moved](https://github.com/Shopify/hydrogen/compare/f117de53c3d65bac0d7d5436bef91128607868ce..a611598f53038efb07c2405e200b8b50eac40f90#diff-60d2f18800416c7d2ebb587a764bce8da4560be2804a396ae7c70575d70a9d0eR18)! Also a good reminder to add [tests](https://github.com/Shopify/hydrogen/compare/f117de53c3d65bac0d7d5436bef91128607868ce..a611598f53038efb07c2405e200b8b50eac40f90#diff-6d3eba7144748467e4ef48ad45b3584dbadcf3506b7bc8dd91387995ff81c6d8R13).